ChangeLog

```diff
@@ -2053,4 +2053,4 @@
 
 - initial version
 
-<!-- vim: syntax=Markdown -->
+<!-- vim: syntax=Markdown: set ft=markdown: -->
```

Comment:
Maybe use the full form of the setting - `filetype` - rather than an abbreviation?

Reply:
(Could do with `filetype=cfg` in `Jamulus.pro`, too.)

Reply:
> (Could do with `filetype=cfg` in `Jamulus.pro`, too.)

Good idea. Though I'll do that separately from this bug fix.

Reply:
> (Could do with `filetype=cfg` in `Jamulus.pro`, too.)

`cfg` is an ok generic filetype for `Jamulus.pro`, but better is to install the syntax and filetype detect files from https://github.com/artoj/qmake-syntax-vim